onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -594,5 +594,28 @@ def testOrtExecutionMode(self):
         opt.execution_mode = onnxrt.ExecutionMode.ORT_PARALLEL
         self.assertEqual(opt.execution_mode, onnxrt.ExecutionMode.ORT_PARALLEL)
 
+    def testLoadingSessionOptionsFromModel(self):
+        try:
+            os.environ['ORT_LOAD_CONFIG_FROM_MODEL'] = str(1)
+            sess = onnxrt.InferenceSession(self.get_name("model_with_valid_ort_config_json.onnx"))
+            session_options = sess.get_session_options()
+            
+            self.assertEqual(session_options.inter_op_num_threads, 5)
+            
+            self.assertEqual(session_options.intra_op_num_threads, 2)
+
+            self.assertEqual(session_options.execution_mode, onnxrt.ExecutionMode.ORT_PARALLEL)
+
+            self.assertEqual(session_options.graph_optimization_level, 3)
```

Comment:
3 should be onnxrt.GraphOptimizationLevel.ORT_ENABLE_ALL

Reply:
Thanks- I ll fix it. Looks like that is causing build to break.

Reply:
Fixed, thanks. I think this breaking the build.